tests/foundry/test/StoreTest.t.sol

```diff
@@ -12,22 +12,31 @@ contract Store {
 
 contract StoreTest is Test {
 
-    Store myStore;
-
-    function setUp() public {
+    function testAccesses() public {
+		Store myStore= new Store();
         vm.record();
-        myStore = new Store();
-    }
 
-    function testAccesses() public {
         (bytes32[] memory reads, bytes32[] memory writes) = vm.accesses(address(myStore));
         assertEq(reads.length, 1);
         assertEq(writes.length, 1);
     }
 
     function testStoreLoad() public {
+		Store myStore= new Store();
+
         vm.store(address(myStore), bytes32(uint256(0)), bytes32(uint256(31337)));
         bytes32 testNumber = vm.load(address(myStore), bytes32(uint256(0)));
         assertEq(uint256(testNumber), 31337); // 31337
```

Comment:
What's going on with the formatting here? Make sure to do self-review on GitHub.

Reply:
Fixed